lua/conform/formatters/stylish-haskell.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://hackage.haskell.org/package/stylish-haskell",
+    description = "A simple Haskell code prettifier",
+  },
+  command = "stylish-haskell",
+  args = { "$FILENAME" },
```

Comment:
From the docs, it looks like `stylish-haskell` supports formatting via stdin & stdout. Do you need this arg?